ecl/hqlcpp/hqlres.cpp

```diff
@@ -99,6 +99,31 @@ static void loadResource(const char *filepath, MemoryBuffer &content)
     read(fio, 0, (size32_t) f->size(), content);
 }
 
+static void queryJfrog(const char *localpath, IPropertyTree &item)
```

Comment:
A duplicate of the code above?  It should only be in one place.

Reply:
Removed duplicate code and included hqlmanifest header file.